source/_integrations/tedee.markdown

```diff
@@ -0,0 +1,34 @@
+---
+title: tedee
+description: Instructions on how to integrate your Tedee lock with Home Assistant.
+ha_release: 2024.1
+ha_category:
+  - Lock
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_domain: tedee
+ha_platforms:
+  - lock
+ha_codeowners:
+  - '@zweckj'
+  - '@patrickhilker'
+ha_integration_type: integration
+---
+
+This integration interacts with your [tedee](https://tedee.com) locks, by communicating with the tedee bridge through HTTP. **The bridge is a requirement for adding your locks through this integration.**
+The integration will communicate with your lock locally, so you need to have the local API enabled.
```

Comment:
```suggestion
This integration interacts with your [Tedee](https://tedee.com) locks by communicating with the Tedee bridge through HTTP. The integration will communicate with your lock locally.

## Pre-requisites

- You will need the bridge for adding your locks using this integration.
- You need to have the local API enabled.
```

Reply:
tedee as brand is all lowercase

Reply:
That was my first thought too... however, that seems to be mostly logo, not textual.

From their own homepage: <https://tedee.com/>

![CleanShot 2023-12-29 at 18 29 37@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/8068d799-7333-4855-b0b3-e249087b44ba)

![CleanShot 2023-12-29 at 18 29 46@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/2419ae27-06bf-441d-a848-b116410397b5)

![CleanShot 2023-12-29 at 18 29 55@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/b3a08f7a-1caa-46bc-8c9e-a388beff61e9)

![CleanShot 2023-12-29 at 18 30 03@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/83ebd9fb-c3c4-499d-b8ec-66232d74b63a)

![CleanShot 2023-12-29 at 18 30 21@2x](https://github.com/home-assistant/home-assistant.io/assets/195327/df905491-9e37-46b9-aa4b-4439c42e4308)

Reply:
Huh. The more you know. I was 100% sure they wrote it lowercase.